test/test_wahba_esoq.py

```diff
@@ -58,7 +60,6 @@ def test_davenport_eigenvalues(self):
                         [1.0, 0.0],
                         [0.0, 1.0]])
         B = pq_esoq.attitude_profile_matrix(obs = obs, ref = ref)
-        irot = pq_esoq.sequential_rotation(B)
```

Comment:
This line is needed. It does an in-place rotation on `B`.

Reply:
so it should just be `pq_esoq.sequential_rotation(B)`? the irot variable isn't used. Also curious why it passes tests with an essential rotation removed, but that's a differnt question.

Reply:
I suggest an additional line that verifies that `irot` is not None. `self.assertNotEqual(irot, None)` maybe?